docs/integrations/pack-docs.md

```diff
@@ -0,0 +1,38 @@
+---
+id: pack-docs
+title: Pack Documentation
+---
+
+:::note
+These guidelines are mainly for packs containing some use case, not a single integration.
```

Comment:
@kirbles19 I want to explain here that these guideline are mainly for packs with use cases/meat on them. Packs with single integration don't need/require to write pack documentation

Reply:
Note sure we need this.

Reply:
I removed this in my commit.